coil-video/src/main/java/coil3/video/imageRequests.kt

```diff
@@ -10,6 +10,32 @@ import coil3.getExtra
 import coil3.request.ImageRequest
 import coil3.request.Options
 
+// region videoFrameIndex
+
+/**
+ * Set the the frame index to extract from a video.
+ *
+ * When both [videoFrameIndex] and other videoFrame-prefixed properties are set,
+ * [videoFrameIndex] will take precedence.
+ */
+fun ImageRequest.Builder.videoFrameIndex(frameIndex: Int) = apply {
```

Comment:
Let's annotate these methods with `@RequiresApi(28)` since they need API 28 to work.

Reply:
Sure thing. I've added it to the README too.